.github/workflows/ci-cd.yml

```
@@ -0,0 +1,350 @@
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Deploy backend to EC2
+        env:
+          BACKEND_IMAGE: ${{ needs.build-and-push.outputs.backend_image }}
+          BACKEND_HOST: ${{ secrets.BACKEND_EC2_HOST }}
+          SSH_PRIVATE_KEY: ${{ secrets.EC2_SSH_PRIVATE_KEY }}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:4:61: Double quote to prevent globbing and word splitting

(shellcheck)